src/Python/qsharp-core/qsharp/serialization.py

```diff
@@ -17,11 +25,17 @@ def map_tuples(obj):
         result = {
             '@type': 'tuple'
         }
-        for i in range(len(obj)):
+        
+        # For tuples of more than 7 items, the .NET type is ValueTuple<T1,T2,T3,T4,T5,T6,T7,TRest>.
+        # Items beyond Item7 must be nested inside a key called "Rest".
+        maxTupleLength = 7
```

Comment:
PEP8 recommends using `snake_case`
https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

Reply:
Good catch! I think the comment here about .NET types must have kicked my mind out of Python mode... 😄